package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "format": "prettier --write .",
-    "seed": "node --env-file=.env ./dataScripts/fetchAllRepos.js"
+    "seed": "node --env-file=.env ./utils/fetchAllRepos.js"
```

Comment:
I think this should be `./utils/githubApi/fetchAllRepos.js`

Reply:
good spot